apps/src/templates/sectionsRefresh/SectionsSetUpContainer.jsx

```diff
@@ -156,7 +158,7 @@ export default function SectionsSetUpContainer({
     }
   };
 
-  const saveSection = (section, createAnotherSection) => {
+  const saveSection = (section, createAnotherSection, coteachersToAdd) => {
```

Comment:
Would it be possible/better to put the coteachers as another child element of the `section` object instead of adding a separate parameter here, or does that make life more difficult elsewhere?

Reply:
I don't want to do that because the section object is directly parsed from the server and the coteachersToAdd has not been sent to the server yet. I think it's easier to tell what they do if they are separate objects.